src/templates.ts

```diff
@@ -87,6 +120,29 @@ export const downloadTemplates =
     }
   }
 
+const enrichWithPrePendingHeader =
+  (mustacheRenderedContent: string, template: Template, codeowners: string) =>
+  (log: Logger): string => {
+    const templateExtension = template.destinationPath.split('.').pop() as string
```

Comment:
Is this cast necessary? 
```suggestion
    const templateExtension = template.destinationPath.split('.').pop()
```